apps/examples/testcase/le_tc/network/itc_net_dup.c

```diff
@@ -0,0 +1,104 @@
+/****************************************************************************
+*
+* Copyright 2017 Samsung Electronics All Rights Reserved.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+* either express or implied. See the License for the specific
+* language governing permissions and limitations under the License.
+*
+****************************************************************************/
+
+///@file itc_net_dup.c
+///@brief Test Case Example for dup() API
+
+#include <sys/socket.h>
+#include "tc_internal.h"
+
+/**
+* @testcase         :itc_net_dup_p_multiple_time
+* @brief            :clone a file descriptor to arbitrary descriptor
+* @scenario         :open socket and clone a file descriptor to arbitrary descriptor multiple times
+* @apicovered       :dup()
+* @precondition     :open socket()
+* @postcondition    :close socket descriptor
+*/
+static void itc_net_dup_p_multiple_time(void)
+{
+	int index;
+	int sock;
+	int ret;
+	int fd;
+
+	for (index = 0; index < 10; index++) {
```

Comment:
how about to assign the number of repeat times by Kconfig or macro?

Reply:
@pillip8282 
Fixed review comments. Used macro to assign number of repeat times.
Please review